### DISCUSSION TITLE
How to match slider limits to zoomed plot limits ?

### DISCUSSION THREAD

**Discussion by auberjine** [JuliaPlots/Makie.jl#4194] [Q&A]
Hi ! 
My problem is probably very simple to fix but I couldn't find how to get the limits of the displayed plot when using the mouse zoom
I was thinking of using Nodes in the slider range and lift them with the displayed limits of the zoomed plot

The idea is to have a precise control over the parameter of the slider when zooming in (in my case the sliders control a window selecting some points of a heatmap from which I calculate some statistics)

This image below shows the interface when the plot is zoomed in quite a bit, the range of the sliders correspond to the dimension of the initial plot so it's a bit tricky to set the sliders precisely for tiny windows

![zoomed](https://user-images.githubusercontent.com/36954632/96005275-95dc4700-0e3c-11eb-8587-79548ede377a.png)

**Comment by jkrumbiegel** [comment]
The current limits are in `ax.limits`. Other than that, you could use one slider to set the mean location, and the other for the zoom range? But what do you gain from this over using the normal mouse zoom I wonder?

**Comment by auberjine** [comment]
ax.limits doesn't seem to follow the displayed limits when zoomed in
I just use the mouse zoom to see what I'm doing (because the heatmap is quite big), the sliders are where to control the parameters of the box selection
